squirrels/_api_routes/auth.py

```
@@ -53,44 +61,36 @@ class AddUserModel(UserInfoModel):
             )
 
         # User info endpoint
-        @auth_router.get("/userinfo", description="Get the authenticated user's fields", tags=["Authentication"])
-        async def get_userinfo(user: RegisteredUser | GuestUser = Depends(self.get_current_user)) -> UserInfoModel:
+        user_session_path = '/user-session'
+
+        @auth_router.get(user_session_path, description="Get the authenticated user's fields", tags=["Authentication"])
+        async def get_user_session(
+            request: Request, user: RegisteredUser | GuestUser = Depends(self.get_current_user)
```

Ruff (0.14.10)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)